Modules/Tokens/src/Tokens.Infrastructure/Persistence/IServiceCollectionExtensions.cs

```diff
@@ -18,13 +18,17 @@ public static void AddPersistence(this IServiceCollection services, Action<Persi
     public static void AddPersistence(this IServiceCollection services, PersistenceOptions options)
     {
         services.AddDatabase(options.DbOptions);
-        services.AddBlobStorage(options.BlobStorageOptions);
-        services.AddRepositories(options.BlobStorageOptions);
+
+        if (options.BlobStorageOptions != null)
+        {
+            services.AddBlobStorage(options.BlobStorageOptions);
+            services.AddRepositories(options.BlobStorageOptions);
```

Comment:
Now you only add the repositories if there is a blob storage configuration. So if we remove it, there will be a lot of errors.